sunpy/map/maputils.py

```diff
@@ -102,6 +103,25 @@ def solar_angular_radius(coordinates):
     return np.arctan(coordinates.rsun / coordinates.observer.radius)
 
 
+def sample_at_coords(smap, coordinates):
+    """
+    Samples the data in a map at given series of coordinates.
```

Comment:
We should probably mention here that it's effectively using nearest neighbour interpolation, because it's array indexing?